### PR TITLE
Add handling for rich-messages and events.

### PR DESCRIPTION
**Pull request by d0t15t**

### CODE REVIEW

**Comment by gambry** [comment]
Apparently travis-ci.org is not usable for builds anymore, so please allow me few days to migrate to travis-ci.com (or anything similar).

This also requires tests.

File: src/Model/Webhook/Response.php [hunk 1]
```diff
@@ -126,14 +127,28 @@ public function createContextFromSession($name, array $data = []) {
...
-        // Append fulfillmentMessages, if any.
+        $json = [];
+        if ($text = $this->getFulfillment()->getText()) {
+          $json['fulfillmentText'] = $this->getFulfillment()->getText();
+        }
         if ($messages = $this->getFulfillment()->getMessages()) {
-            $json['fulfillmentMessages'] = array_map(function($message) {
-                return $message->jsonSerialize();
-            }, $messages);
+          $json['fulfillmentMessages'] = [
```
Comment (gambry) [inline src/Model/Webhook/Response.php]: This shouldn't be hardcoded. We should have a Message class implementing `JsonSerializable()` so the logic in this block is delegated to Messages. Same for Event.

**Review by gambry — CHANGES_REQUESTED** [review]

**Comment by gambry** [comment]
@d0t15t I've got Travis CI working again.

**Comment by d0t15t** [comment]
@gambry Thanks, I've added the classes.

**Comment by gambry** [comment]
@d0t15t I'm closing this PR in favour of #3 .

Let's continue the conversation in there.
Thanks!

**Pull request closed without merging**